package.json

```diff
@@ -26,10 +26,10 @@
     "bindings": "~1.2.1",
     "commander": "^2.8.1",
     "glob": "^5.0.14",
-    "request": "=2.81.0",
-    "node-gyp": "~3.6.2",
+    "node-gyp": "^3.7.0",
     "readable-stream": "^2.1.4",
-    "tap": "~0.7.1",
+    "request": "^2.87.0",
```

Comment:
FWIW, I have no idea why we depend on `request`.  We don't use it anywhere.

node-gyp is another dependency that should probably go.  I recommend (I also maintain node-gyp) to use the version of node-gyp that comes with node/npm, not install your own.

Reply:
Hey I found we have some redundant dependency...
"node-gyp is another dependency that should probably go ... " -> I do not know what you meaning about ?